server/src/main/java/p2/webservice/UserWebService.java

```diff
@@ -0,0 +1,131 @@
+package p2.webservice;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.log4j.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import p2.model.User;
+import p2.service.impl.UserService;
+import p2.util.SessionVariableManager;
+
+public class UserWebService {
+
+	private static Logger logger = Logger.getLogger(UserWebService.class);
+
+	public static void login(HttpServletRequest request, HttpServletResponse response) {
+
+		String email = request.getParameter("email");
+		String password = request.getParameter("password");
+
+		User user = UserService.findByEmailANDPassword(email, password);
+		try {
+			if (user != null) {
+				SessionVariableManager.addLoggedInUser(request, user);
+				logger.info("User Id: " + user.getId() + " Logged to the sysytem");
+				response.getWriter().append("Successfully Logged to the system").close();
+
+			} else {
+				response.getWriter().close();
+			}
+
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static  void getLoggedInUser(HttpServletRequest request, HttpServletResponse response) {
+
+		ObjectMapper om = new ObjectMapper();
+
+		try {
+			String json = om.writeValueAsString(SessionVariableManager.getLoggedInUser());
+			response.getWriter().append(json).close();
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void logout(HttpServletRequest request, HttpServletResponse response) {
+		try {
+			SessionVariableManager.removeLoggedInUser();
+			response.getWriter().close();
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void insert(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
we need simple validation of these parameters (all of them because of postman testing)